### ISSUE TITLE
Improve keyboard navigation for modals

### ISSUE THREAD

**Issue by cstarcher** [TexasDigitalLibrary/Vireo#1660]
Relevant Components:

- All Modals

Description
When focus is on a modal, the “esc” key should exit the modal.

People using the keyboard often use hotkeys such as “esc” to navigate, and when the element does not behave as expected, confusion can occur.

People with vision and cognitive issues often use solely the keyboard to navigate.

An event listener for the “esc” key, resulting in the modal closing, would improve the accessibility of this element.

Solution
Make sure the modal acts as can be reasonably expected by a user - specifically, the “esc” key should exit the modal.

**Comment by jcreel** [comment]
Will this require a modification to the Weaver modal?

**Comment by wwelling** [comment]
https://getbootstrap.com/docs/3.4/javascript/#modals-options

**Comment by wwelling** [comment]
https://github.com/TAMULib/Weaver-UI-Core/blob/2.x/app/views/modalWrapper.html

**Comment by wwelling** [comment]
@jcreel your intuition may be correct. I was unable to find solution without Weaver change. It actually appears that Bootstrap 3 JavaScript and/or AngularJS is no longer functioning as expected. We have had this implemented for some time and is in use throughout Vireo. From what I can tell, this is a regression in Weaver.

Here is a solution to resolve, https://github.com/TAMULib/Weaver-UI-Core/pull/232
